build.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -e
+
+while getopts t:r:b:v:c: flag
+do
+    case "${flag}" in
+        c) config=${OPTARG};;
+    esac
+done
+
+config=$([ -z "$config" ] && echo "./config/prod.json" || echo "$config")
+
+# Workaround for the src folder
+mv -f lib/ds-test/src lib/ds-test/contracts
+mv -f lib/subfactory/lib/ds-test/src lib/subfactory/lib/ds-test/contracts
+mv -f lib/oracle/lib/ds-test/src lib/oracle/lib/ds-test/contracts
+mv -f lib/calculator/lib/ds-test/src lib/calculator/lib/ds-test/contracts
+
+export DAPP_TEST_TIMESTAMP=1622483493
+export DAPP_TEST_NUMBER=12543537
+export DAPP_SOLC_VERSION=0.6.11
+export DAPP_SRC="contracts"
+export DAPP_LINK_TEST_LIBRARIES=0
+export DAPP_STANDARD_JSON=$config
+
+dapp --use solc:0.6.11 build
+
+# Workaround for the src folder
+mv -f lib/ds-test/contracts lib/ds-test/src
+mv -f lib/subfactory/lib/ds-test/contracts lib/subfactory/lib/ds-test/src
+mv -f lib/oracle/lib/ds-test/contracts lib/oracle/lib/ds-test/src
+mv -f lib/calculator/lib/ds-test/contracts lib/calculator/lib/ds-test/src
```

Comment:
We're going to put all submodules in `modules` directory to prevent opinionated dapp from the weird stuff it assumes should be in `lib`.